pkg/cloud/scope/cluster.go

```diff
@@ -170,7 +170,7 @@ func (s *ClusterScope) AdditionalTags() infrav1.Tags {
 		s.AWSCluster.Spec.AdditionalTags = infrav1.Tags{}
 	}
 
-	return s.AWSCluster.Spec.AdditionalTags
+	return infrav1.Tags(s.AWSCluster.Spec.AdditionalTags).DeepCopy()
```

Comment:
Isn't AdditionalTags already of `Tags` type? If not, we should make it so, if it's not a breaking change

Reply:
Both the Machine and Cluster were like that, so I didn't touch it. I don't know if it's breaking or not – I'll give it a try and report back.

Reply:
Once I figured out what package I was in, everything seemed to work out fine.